### DISCUSSION TITLE
Do you need to uninstall original Lua before installing Pluto?

### DISCUSSION THREAD

**Discussion by Its-Kenta** [PlutoLang/Pluto#114] [Q&A] [answered]
Just a curiosity question here, I’m unsure if Pluto has its own extension name like most supersets or does Pluto overwrite the .Lua format?

Also amazing work so far! I absolutely adore what you’re doing, this is by far my most favourite Lua I’ve seen so far and makes me want to go back into using Lua.

**Comment by well-in-that-case** [comment, marked as answer]
When you compile Pluto, it'll create it's own binaries:
- `libpluto.a`
- `pluto.exe`
- `plutoc.exe`

So, you can (i.e) add these to your PATH and it'll work seamlessly with pre-existing Lua installations.

Also, thanks for the praise.

**Reply by Its-Kenta** [reply]
Thank you, then I understand that to normally run Pluto code it would be with Pluto? e.g pluto mycode.lua ? Or still Lua as usual and vice versa?

**Reply by well-in-that-case** [reply]
Yeah, it would be `pluto mycode.lua`
